resources/lang/en/internet.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+return [
+    "your_registered_devices" => "Your registered devices",
+    "admin.manage_registered_devices" => "Manage registered devices",
+    "mac_address" => "MAC address",
+    "comment" => "Comment",
+    "state" => "State",
+    "requested" => "Requested",
+    "approved" => "Approved",
+    "rejected" => "Rejected",
+    "add" => "Add device",
+    "delete" => "Delete",
+    "cancel" => "Cancel",
+    "confirm_delete" => "Are you sure you want to delete?",
+    'error' => 'Error',
+    'ajax_error' => 'Server error! Please refresh the site!',
+    'ok' => 'OK',
+    'username' => 'Username',
+    'approve' => 'Approve',
+    'reject' => 'Reject',
+    'request' => 'Undo',
```

Comment:
This looks even worse in english. Maybe the key should be changed?
cc @vendi95

Reply:
What's going on? Request -> undo what?

Reply:
ping @kiscs